src/cloud/tags.yml

```diff
@@ -55,20 +55,54 @@ tags:
         - [Manage buckets]({{% INFLUXDB_DOCS_URL %}}/organizations/buckets/)
   - name: Invokable Scripts
     description: |
-      Manage and execute scripts as API endpoints in InfluxDB.
+      Store, manage, and execute scripts in InfluxDB.
 
-      An API Invokable Script assigns your custom Flux script to a new
-      InfluxDB API endpoint for your organization.
-      Invokable scripts let you execute your script as an HTTP request to the endpoint.
+      An invokable script stores your custom Flux script and lets you execute
+      the script from tasks and API requests.
+      To use invocation parameters, include `params` references in your script--for example: `params.myparameter`.
+      When you invoke the script, you pass `params` with key-value pairs for your parameters.
+      Then, InfluxDB executes your script and returns the result.
 
-      Invokable scripts accept parameters.
-      Add parameter references in your script as `params.myparameter`.
-      When you `invoke` your script, you send parameters as key-value pairs in the `params` object.
-      Then, InfluxDB executes your script with the key-value pairs as arguments, and returns the result.
+      To create a script, you can use the [`POST /api/v2/scripts`](#operation/PostScripts) endpoint.
+      After you create a script, extract the script `id` from the response.
+      With the script ID, you can [invoke the script as an API endpoint](#invoke-a-script-as-an-api-endpoint)
+      or [add the script to a task](#add-a-script-to-a-task).
+
+      #### Invoke a script as an API endpoint
+
+      Once you create a script, InfluxDB generates an [`/api/v2/scripts/SCRIPT_ID/invoke` endpoint](#operation/PostScriptsIDInvoke) for your
```

Comment:
should we have another procedure above to create a script? Either way, think it'd be helpful to be explicit on step for every action they need to take

**Create a script**
1. Use the [`POST /api/v2/scripts`](#operation/PostScripts) endpoint, and do the following:
- Include `params` references in your script--for example, `params.myparameter`
-  (not sure if this is part of creating the script or invoking the script?) Pass `params` with key-value pairs in the response
InfluxDB generates an [`/api/v2/scripts/SCRIPT_ID/invoke` endpoint](#operation/PostScriptsIDInvoke) for your organization.